docker/postgresql/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM postgres:9.2
```

Comment:
🤔 Do we know what version is running in production?
(I've been working with `9.4.17`, assuming 9 is fairly stable.)

Reply:
CentOS (and RHEL) use this version line as default, in true RHEL fashion, long after it has been EOLed by the maintainers. I don't know for sure that that's what CDOS is using, but that's why this is so old. We might be able to infer this information from some data they've recently given us, though.